### PR TITLE
Add Quest Extension

### PR DESCRIPTION
**Pull request by zeekrey**
## Description

Hey 👋🏻

**Quest** is a simple extension that allows you to create and work on checklists. Based on [The Checklist Manifesto](https://en.wikipedia.org/wiki/The_Checklist_Manifesto) you know checklists can save lives and make you more productive by improving the quality of repetitive tasks. **Quest** lets you create, edit and delete checklists (quests). You can easily start working on them and keep track of the progress.

### How it works

1. Call `Show Quests`.
2. Add a Quest (Checklist) by pressing `CMD+N`.
3. Give it a name and press `CMD+Enter` to add tasks. Finally add the quest by pressing `CMD+K` → `Create Quest`.
4. Now select the quest and press `Enter` to start working on it whenever you need to do the task. Check off tasks by pressing `Enter`.
5. Quests in progress will be listed at the top of the `Show Quests`-Command.
6. Once you're done with a quest, export a markdown report by pressing `CMD+T` → `Export Markdown`.

## Screencast

https://user-images.githubusercontent.com/34539149/223559964-980290a3-05d5-4d8a-89d1-626614a03ec4.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/quest/src/actions/deleteQuest.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Action, Icon } from "@raycast/api";
+
+function DeleteQuestAction(props: { onDelete: () => void }) {
+  return (
+    <Action
+      icon={Icon.Trash}
+      title="Delete Quest"
```
Comment (pernielsentikaer) [inline extensions/quest/src/actions/deleteQuest.tsx:7]:
```suggestion
      title="Delete Quest"
      style={Action.Style.Destructive}
```
Could be nice to have a [confirmAlert](https://developers.raycast.com/api-reference/feedback/alert#confirmalert) somewhere too 😊

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

* Could you move the folder `fixtures` to `src`

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/quest/src/actions/deleteQuest.tsx (hunk 1 above)
Comment (zeekrey) [reply extensions/quest/src/actions/deleteQuest.tsx:7]: Added in d026d26df25cf65fb2ebc1bd3f1b4aed94c5aff8

File: extensions/quest/src/actions/editQuest.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Action, Icon } from "@raycast/api";
+import CreateQuestForm from "../components/createQuest";
+import type { Quest } from "../types";
+
+function EditQuestAction(props: { onCreate: (quest: Omit<Quest, "id">) => void; quest: Quest }) {
+  return (
+    <Action.Push
+      icon={Icon.Pencil}
+      title="Edit Quest"
```
[the remark this answers is not in the document]
Comment (zeekrey) [reply extensions/quest/src/actions/editQuest.tsx:9]: Added in b3b0b4501e2ef681faa007f25c8eeb70709728cd

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋

Looks good to me, approved 🔥

**Pull request merged**